### DISCUSSION TITLE
Network configuration

### DISCUSSION THREAD

**Discussion by Gulshan45** [telekom-security/tpotce#1394] [Q&A]
When I tried to install t-pot using post install method on my laptop with Ubuntu. After successful installation and rebooting the system my network connection lost that is I cannot connect my LAN port and my settings are also disabled. I was not even able to ping my IP. How should I resolve this issue so that I can ping my IP and able to open the kibana?

**Comment by t3chn0m4g3** [comment]
T-Pot 22.04 does not support Ubuntu.

**Reply by Gulshan45** [reply]
I have post install it on Debian 12 too. But this time again my ethernet get disconnected and I'm not able to login to my laptop. I'm stuck at the starting profile window.

**Reply by Gulshan45** [reply]
Which version of t pot is suitable for Ubuntu 22.04?

**Reply by t3chn0m4g3** [reply]
Only the Technical Preview #1325.

**Reply by Gulshan45** [reply]
Thank you. I'll try that

**Reply by Gulshan45** [reply]
i have installed debian 11.7 and then performed the post installation method. the installation is compelete and according to the documentation i can ping my ip but cannot perform ssh. But the problem here is that i am not able to open the GUI that is kibana from my IP. like by typing   ip:64297  or 64295
